### PR TITLE
Leave review POST request

### PR DESCRIPTION
**Pull request by samliu10**
### Summary 

This pull request is for the POST request that submits a review to the backend. The review must be of type `Review` specified in `db-types.ts`. 

### Test Plan 
Run `yarn start` in `backend` to start a local dev server. Test the endpoint using a POST request in Postman. The Firebase database will update with the new review in real-time.

### CODE REVIEW

File: backend/src/app.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import express, { Express } from 'express';
-import bodyParser from 'body-parser';
 import cors from 'cors';
 import { db } from './firebase';
 import { Section } from './firebase/types';
+import { Review } from '../../common/types/db-types';
 
 const app: Express = express();
+const reviews = db.collection('reviews');
```
Comment (ptwu) [inline backend/src/app.ts:8]: make this more indicative that it's a collection (`reviews` is kind of generic)

**Review by ptwu — COMMENTED** [review]
looks good, just one naming nit

File: backend/package.json [hunk 2]
```diff
@@ -11,18 +11,18 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@firebase/testing": "^0.20.11",
```
Comment (Enochen) [inline backend/package.json:14]: According to [the npm page](https://www.npmjs.com/package/@firebase/testing), this package is deprecated

File: backend/src/app.ts [hunk 3]
```diff
@@ -29,4 +30,11 @@ app.get('/', async (req, res) => {
   res.status(200).send(JSON.stringify(faqs));
 });
 
+app.post('/new-review', async (req, res) => {
+  const doc = reviews.doc();
+  const review: Review = req.body;
```
Comment (Enochen) [inline backend/src/app.ts:35]: Not sure if we're at this stage yet but we probably should destructure the properties we want. This prevents us from storing extra/non-schema-related properties that attackers might include in the request body.

Comment (ptwu) [reply backend/src/app.ts:35]: alternatively, you can just do `const review = req.body as Review`

**Review by ptwu — APPROVED** [review]

**Pull request merged**